### PR TITLE
[release] [patch] updated docs to help devs to use it integrating wit…

### PR DESCRIPTION
**Pull request by daniloster**
…h i18next

## Changes

- Added docs related to integration with i18next
- Updated/fixed old docs

## Tests

- [ ] Manually Tested
- [ ] Unit Tested
- [ ] Visually Tested

### CODE REVIEW

**Comment by daniloster** [comment]
@adrai  FYI, here is the doc.

File: docs/GETTING_STARTED.md [hunk 1]
```diff
@@ -6,12 +6,17 @@ The usage of `@daniloster/svelte-i18n` is based on some conventions which can sp
...
+
+Here we are starting development with a simple case, the proper lib is taking care of the internationalization. Basically, the look up for the text related to the json path.
 
 _`src/i18nState.js`_
 
 ```js
+// Either
 import factoryI18nState from '@daniloster/svelte-i18n'
+// or
+// import { factoryI18nState } from '@daniloster/svelte-i18n'
```
Comment (adrai) [inline docs/GETTING_STARTED.md:19]: this or seems like the code line before

File: docs/GETTING_STARTED.md [hunk 2]
```diff
@@ -284,13 +309,116 @@ const i18nState = factoryI18nextState({
...
-| :------------: | :------------------------: | :----------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| clearNamespace | `Function(String): String` | :heavy_check_mark: | The default function strips out initials `src/`, `DEV/` or `dist/.app`. Also, strips extensions (`.js`, `.jsx`, `.svelte`) and converts remaining `/` into `.` |
-|      i18n      |         `i18next`          |                    | It is required, and it is the mechanism to resolve messages                                                                                                    |
-|  persistence   |          `Object`          | :heavy_check_mark: | _(`default: { get: noop, set: noop }`)_ The service to get and persist locale changes                                                                          |
+From static usage to remote (fetching resources on demand), the `i18next` the library will need to have the property `load: 'currentOnly'` as well as the load path.
```
Comment (adrai) [inline docs/GETTING_STARTED.md:314]: the load option depends on how the app is localized... could be also an other option, but ok

**Comment by adrai** [comment]
rest lgtm 👍

File: docs/GETTING_STARTED.md (hunk 1 above)
Comment (daniloster) [reply docs/GETTING_STARTED.md:19]: It is exported as default and named export. the diff is `{`

Comment (adrai) [reply docs/GETTING_STARTED.md:19]: ahh sorry, my eyes are tired

Comment (daniloster) [reply docs/GETTING_STARTED.md:19]: No worries, thanks for the feedback though

**Pull request merged**